kalite/facility/templates/facility/facility_user.html

```diff
@@ -71,7 +71,14 @@ <h3 class="no-margin-top">{{ title }}</h3>
         <form method="post" class="basic-form">
             {% csrf_token %}
 
-            {{ form.as_p }}
+            {% for field in form.visible_fields %}
```

Comment:
Check out the [docs](https://docs.djangoproject.com/en/1.5/topics/forms/#looping-over-hidden-and-visible-fields).